translations/dashboard.fr.yml

```
@@ -1,43 +1,87 @@
 dashboard:
-  Base_urssaf: B. URSSAF
+  aucun_commentaire : Aucun commentaire sur les études durant les 10 derniers jours.
+  book_passation : Book de passations
+  base_urssaf: Bases URSSAF
+  actico: Activité Commerciale
```

Essayons d'éviter les abbréviations personnelles.

```suggestion
  activité_commerciale: Activité Commerciale
```